skimage/morphology/_skeletonize_3d_cy.pyx.in

```diff
@@ -134,26 +134,26 @@ cdef list find_simple_point_candidates(pixel_type[:, :, ::1] img,
         int[::1] Euler_LUT = LUT
 
     # loop through the image
-    # NB: each loop is from 1 to size-1: img is padded from all sides 
-    for p in range(1, img.shape[0] - 1):
-        for r in range(1, img.shape[1] - 1):
-            for c in range(1, img.shape[2] - 1):
+    # NB: each loop is from 1 to size-1: image is padded from all sides 
+    for p in range(1, image.shape[0] - 1):
+        for r in range(1, image.shape[1] - 1):
+            for c in range(1, image.shape[2] - 1):
 
                 # check if pixel is foreground
-                if img[p, r, c] != 1:
+                if image[p, r, c] != 1:
                     continue
 
-                is_border_pt = (curr_border == 1 and img[p, r, c-1] == 0 or  #N
-                                curr_border == 2 and img[p, r, c+1] == 0 or  #S
-                                curr_border == 3 and img[p, r+1, c] == 0 or  #E
-                                curr_border == 4 and img[p, r-1, c] == 0 or  #W
-                                curr_border == 5 and img[p+1, r, c] == 0 or  #U
-                                curr_border == 6 and img[p-1, r, c] == 0)    #B
+                is_border_pt = (curr_border == 1 and image[p, r, c-1] == 0 or  #N
```

Comment:
Hmmm. These lines no longer fit in 80 chars... Maybe replace these comments with a comment at the top?